src/Reflection/Pattern.agda

```diff
@@ -25,9 +25,41 @@ open import Reflection.Argument
 ------------------------------------------------------------------------
 -- Re-exporting the builtin type and constructors
 
-open import Agda.Builtin.Reflection public using (Pattern)
+open import Agda.Builtin.Reflection as Builtin public using (Pattern)
 open Pattern public
 
+------------------------------------------------------------------------
+-- Showing
+
+mutual
+
+  showPatterns : List (Arg Pattern) → String
+  showPatterns [] = ""
+  showPatterns (a ∷ ps) = showArg a ++ showPatterns ps
+    where
+      showRel : Builtin.Relevance → String
```

Comment:
Can we get rid of the `Builtin.` here? Not sure they're adding to the clarity...

Reply:
Good point.